test/runtests.jl

```diff
@@ -5,6 +5,8 @@ using AbstractFFTs: Plan, plan_inv
 using Test
 using LinearAlgebra
 
+@info "FFTW provider: $(fftw_provider)"
```

Comment:
Perhaps I'm missing it, but I couldn't find an easy way to tell what's the provider when running the tests

Reply:
Yeah, I just know to look for the `DCT` tests, which only run on `fftw`.  But this is clearly better.  :)